ssl/t1_lib.c

```diff
@@ -1196,13 +1410,35 @@ int ssl_setup_sig_algs(SSL_CTX *ctx)
             cache[i].enabled = 0;
         EVP_PKEY_CTX_free(pctx);
     }
+
+    /* Now complete cache and tls12_sigalgs list with provider sig information */
+    cache_idx = OSSL_NELEM(sigalg_lookup_tbl);
+    for (i=0; i<ctx->sigalg_list_len; i++) {
+	TLS_SIGALG_INFO si = ctx->sigalg_list[i];
+	cache[cache_idx].name = si.algorithm;
+	cache[cache_idx].sigalg = si.code_point;
+	tls12_sigalgs_list[cache_idx] = si.code_point;
+	cache[cache_idx].hash = OBJ_sn2nid(si.hash_algorithm);
+	cache[cache_idx].hash_idx = ssl_get_md_idx(cache[cache_idx].hash);
+	cache[cache_idx].sig = OBJ_sn2nid(si.algorithm);
+	cache[cache_idx].sig_idx = i+SSL_PKEY_NUM;
```

Comment:
nit: space around `+`

Reply:
So changed in 0ac814f8247d1a5f157eec2b62bce0000c3a2d1e.